.github/workflows/testing.yml

```diff
@@ -86,7 +86,7 @@ jobs:
     - name: Install packages
       run: |
         pip install ${{ matrix.numpy-version }}
-        pip install setuptools wheel cython gmpy2 pytest pytest-xdist pybind11 pytest-xdist mpmath scikit-umfpack scikit-sparse
+        pip install setuptools wheel cython pytest pytest pytest-xdist pybind11 pytest-xdist mpmath gmpy2
```

Comment:
looks like `pytest` is showing up twice; either of us can fix that easily enough